Makefile

```diff
@@ -7,6 +7,9 @@ LINUX:="Linux"
 all: parser.go
 
 test: parser.go
+	mv go.mod1 go.mod
+	mv go.sum1 go.sum
+	trap 'mv go.mod go.mod1;mv go.sum go.sum1' EXIT
```

Comment:
from https://stackoverflow.com/questions/15656492/linux-shell-try-catch-finally it works @tiancaiamao

Reply:
I follow the way of the first answer.